src/_scss/pages/search/results/table/_resultsTableMessageContainer.scss

```diff
@@ -0,0 +1,48 @@
+.results-table-message-container {
+    display: flex;
+    flex-flow: column wrap;
+    position: relative;
+    background-color: #F6F6F6;
```

Comment:
We could use the variable `$color-gray-lightest` here (to ensure color consistency throughout the site)